build/dependency.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o pipefail
+set -e
+set -x
+
+export LC_ALL=C
+
+PWD=$(
+  cd "$(dirname "$0")"/.. || exit
+  pwd
+)
+
+MVN="${PWD}/build/mvn"
+
+DEP_PR="${PWD}/dev/dependencyList.tmp"
+DEP="${PWD}/dev/dependencyList"
+
+function build_classpath() {
+  $MVN install -Pspark-3.3 -DskipTests
+  $MVN dependency:build-classpath -pl master,worker,common -Pspark-3.3 |
```

Comment:
since the master and worker classpath folder is separated, should we separate the dependencyList as well?

Reply:
Actually, the dependencyList is not used now. So maybe it is not a problem.

Reply:
The value of the dependency list is to make sure there is no silent unexpected dependencies change. I guess a lot of big data engineers learned class/jar conflict from Hadoop/Hive -related projects, it's always painful, especially Guava.